packages/@tests/smoke/node/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import '@fluencelabs/js-client.node';
+import '@fluencelabs/js-peer';
```

Comment:
Do you plan for unified package to be called `js-peer`? Or is this just temporary? I expected it to be called `js-client`, I think most people expect this too